app/controllers/abouts_controller.rb

```diff
@@ -0,0 +1,4 @@
+class AboutsController < ApplicationController
+  def show
+  end
+end
```

Comment:
Do we want to be able to show this page without authentication?

Reply:
Good question, not sure

Reply:
If we want to make it accessible without authentication, we should add a link to it on the non authenticated pages as well

Reply:
I agree, this is a good idea. I'll ask Charlie to place it somewhere in the unauthenticated views.

Although I would not consider this to be a blocker for approval.